sentences/en/light_HassLightSet.yaml

```diff
@@ -71,6 +65,23 @@ intents:
           area:
             slot: true
 
+      # Floor support for brightness
+      - sentences:
+          - "[<numeric_value_set>] [the] brightness in <floor> to <brightness>"
+          - "[<numeric_value_set>] [the] brightness of <floor> to <brightness>"
```

Comment:
Use `<in>` which is defined in `_common.yaml`.

Reply:
...and make it optional (e.g. `set the brightness upstairs to 25%`)

Reply:
It's a copy-paste of the current sentences in English for areas.
But we can work a bit but we're not maintainer of that language

Reply:
Understood. I'm not sure how to go about this, as it appears we are digging ourselves in.

For example in `core`, even if the code coverage is bad, you can't make it worse in a PR. I was hoping not to copy whatever was present for `{area}` as that's not necessarily a benchmark.

Leave this PR as you see fit and I'll try to update with the changes I myself proposed.

Reply:
Non no it's up to you to decide. You're the English LL 😉
We just felt bad not adding "something" for English. 

I have an idea, how about we add just ONE sentence for floor support just to state the fact it is supported in English. 

Would that be ok? 
I can remove the copy paste and provide a single sentence.

Reply:
Edit: Done. Will ask you to re-review in a minute